test/development/basic/styled-components-disabled.test.ts

```diff
@@ -0,0 +1,65 @@
+import { join } from 'path'
+import webdriver from 'next-webdriver'
+import { createNext, FileRef } from 'e2e-utils'
+import { NextInstance } from 'test/lib/next-modes/base'
+import { fetchViaHTTP } from 'next-test-utils'
+
+describe('styled-components SWC transform', () => {
+  let next: NextInstance
+
+  beforeAll(async () => {
+    next = await createNext({
+      files: {
+        'next.config.js': new FileRef(
```

Comment:
Is the `next.config.js` being used here? This can be provided via the `nextConfig` field on `createNext` instead which is typed instead.